src/utils/schema-formatter.ts

```
@@ -0,0 +1,178 @@
+    
+    const typeName = (innerType as any)._def?.typeName;
+    
+    switch (typeName) {
+      case 'ZodString':
+        return 'example_string';
+      case 'ZodNumber':
+        return 42;
+      case 'ZodBoolean':
+        return true;
+      case 'ZodArray':
+        const elementExample = getZodTypeExample((innerType as z.ZodArray<any>).element);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)